### PR TITLE
Hemal/modify recommended resources

### PR DESCRIPTION
**Pull request by Hemal1P**
## Overview

To improve upon high volume deployments whose SCIM bridge crashes due to low resources we are increasing 
recommendations. 

## Changes

* Readme.md has been updated with recommended resources for various deployments. 

-----------------------------------------------------------------------

## Checklist

- [x] review the [guide to contributing](https://github.com/1Password/op-scim-helm/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by chasdl** [comment]
Thanks Hemal! I'm thinking we can modify the table values. `2000` is a bit low for the high volume deployment, let's adjust the range a bit to `5000`. Let's do:

| 100 - 5000  |  High Volume Deployment  |
| 5000+  |  Very High Volume Deployment  |

File: README.md [hunk 1]
```diff
@@ -50,20 +59,37 @@ limits:
...
 
-Proposed recommendations for high volume deployments:
+These values can be scaled down again after the high volume deployment.
 
-```yaml
-requests:
-  cpu: 500m
-  memory: 512M
+<details>
+  <summary>High Deployment</summary>
```
Comment (alicethorne-ab) [inline README.md:67]:
```suggestion
  <summary>High Volume Deployment</summary>
```

File: README.md [hunk 2]
```diff
@@ -50,20 +59,37 @@ limits:
...
+
+  limits:
+    cpu: 1000m
+    memory: 1024M
+  ```
+</details>
 
-This proposal is 4x the CPU and 2x the memory of the default values. These values can be scaled down again after the high volume deployment.
+<details>
+  <summary>Very High Deployment</summary>
```
Comment (alicethorne-ab) [inline README.md:81]:
```suggestion
  <summary>Very High Volume Deployment</summary>
```

**Review by alicethorne-ab — APPROVED** [review]
A couple of comments, but otherwise, this LGTM. :shipit:

File: README.md (hunk 1 above)
Comment (Hemal1P) [reply README.md:67]: good catch.

File: README.md [hunk 3]
```diff
@@ -50,20 +59,37 @@ limits:
   cpu: 250m
   memory: 512M
 ```
+</details>
 
-Proposed recommendations for high volume deployments:
+These values can be scaled down again after the high volume deployment.
```
[the remark this answers is not in the document]
Comment (chasdl) [reply README.md:64]: Nevermind I see we mention this in the second paragraph under Resource Recommendations

**Review by chasdl — APPROVED** [review]

**Review by chasdl — APPROVED** [review]

**Pull request merged**